src/test/java/hobbylist/logic/commands/FindCommandTest.java

```diff
@@ -3,69 +3,87 @@
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
 
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.function.Predicate;
 
 import org.junit.jupiter.api.Test;
 
 import hobbylist.model.Model;
 import hobbylist.model.ModelManager;
 import hobbylist.model.UserPrefs;
+import hobbylist.model.activity.Activity;
+import hobbylist.model.activity.DateMatchesGivenDatePredicate;
 import hobbylist.model.activity.NameOrDescContainsKeywordsPredicate;
+import hobbylist.model.activity.RatingMatchesGivenValuePredicate;
 import hobbylist.testutil.TypicalActivities;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
-    private Model expectedModel = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
+    private final Model model = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
 
     @Test
     public void equals() {
-        NameOrDescContainsKeywordsPredicate firstPredicate =
+        NameOrDescContainsKeywordsPredicate firstKeywordPredicate =
                 new NameOrDescContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameOrDescContainsKeywordsPredicate secondPredicate =
+        DateMatchesGivenDatePredicate firstDatePredicate = new DateMatchesGivenDatePredicate("2022-01-01");
+        RatingMatchesGivenValuePredicate firstRatingPredicate = new RatingMatchesGivenValuePredicate(3);
+        NameOrDescContainsKeywordsPredicate secondKeywordPredicate =
                 new NameOrDescContainsKeywordsPredicate(Collections.singletonList("second"));
+        DateMatchesGivenDatePredicate secondDatePredicate = new DateMatchesGivenDatePredicate("2021-02-02");
+        RatingMatchesGivenValuePredicate secondRatingPredicate = new RatingMatchesGivenValuePredicate(2);
 
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
+        FindCommand findFirstCommand = new FindCommand(firstKeywordPredicate,
+                                                        firstDatePredicate,
+                                                        firstRatingPredicate);
+        FindCommand findSecondCommand = new FindCommand(secondKeywordPredicate,
+                                                        secondDatePredicate,
+                                                        secondRatingPredicate);
 
         // same object -> returns true
-        assertTrue(findFirstCommand.equals(findFirstCommand));
+        assertEquals(findFirstCommand, findFirstCommand);
```

Comment:
Could this be the reason that code coverage fell because `FindCommand::equals` is not explicitly tested but implicitly tested?

Reply:
Let me test and see if this is the reason making the coverage smaller.